presto-main/src/main/java/com/facebook/presto/operator/scalar/BitwiseFunctions.java

```diff
@@ -52,6 +52,14 @@ public static long bitCount(@SqlType(StandardTypes.BIGINT) long num, @SqlType(St
         return Long.bitCount(num & mask);
     }
 
+    @Description("count number of trailing zero bits in 2's complement representation")
+    @ScalarFunction
+    @SqlType(StandardTypes.BIGINT)
+    public static long numTrailingZeros(@SqlType(StandardTypes.BIGINT) long num)
```

Comment:
This adds a `num_trailing_zeros(bigint)` function to Presto in addition to the functions mentioned in the PR description.

A quick unit test would be good here.